enos/modules/worker/main.tf

```diff
@@ -203,3 +221,19 @@ resource "enos_boundary_start" "worker_start" {
     }
   }
 }
+
+resource "enos_remote_exec" "update_directory" {
+  depends_on = [enos_boundary_start.worker_start]
+  count      = var.recording_storage_path == "" ? 0 : 1
+
+  inline = [
+    "sudo chmod -R 755 ${var.recording_storage_path}",
+    "sudo chown boundary:boundary ${var.recording_storage_path}"
```

Comment:
Can this not be done as part of the `create_directory`?

Reply:
When I tried it, the `boundary` user didn't exist until after the worker started.

Reply:
What is the resource that creates the `boundary` user? Can that either:
- be separated from starting the worker so this can depend just on the existence of the user, and then starting the worker can depend on this.
- or update that resource to be the thing that can optionally create this directory after it has created the user?

Reply:
Exploring an option to create the directory after it creates the user: https://github.com/hashicorp/enos-provider/pull/189. Need to work with the enos team on validating the implementation.

Reply:
Got that added to `enos-provider`